src/appsignal/helpers.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import annotations
+
+import json
+import logging
+from typing import TYPE_CHECKING, Any
+
+from opentelemetry import trace
+
+
+if TYPE_CHECKING:
+    from opentelemetry.trace import Span
+
+logger = logging.getLogger("appsignal")
+
+
+def set_attribute(attribute: str, value: Any, span: Span | None = None) -> None:
+    span = span or trace.get_current_span()
+    # TODO: `trace.get_current_span` will return an INVALID_SPAN
+    # if there is no current span. Should we log something about
+    # it? The Node.js integration does log something about it.
+    #
+    # To detect if it's an invalid span, it seems we would have
+    # to do something like:
+    # `span.get_span_context().is_valid`
+    span.set_attribute(attribute, value)
+
+
+def set_serialised_attribute(
```

Comment:
Shall we make this private with an underscore prefix?

```suggestion
def _set_serialised_attribute(
```

Reply:
Yes! 👍  Same for `set_attribute` and `set_prefixed_attribute`.